agg.pretty/RJMG16X.kicad_mod

```diff
@@ -0,0 +1,37 @@
+(module RJMG16X (layer F.Cu) (tedit 592324CE)
+  (descr http://www.farnell.com/datasheets/71713.pdf)
+  (fp_text reference REF** (at 0 -12.03) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value RJMG16X (at 0 16.1) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -8.4 -10.92) (end 8.4 -10.92) (layer F.SilkS) (width 0.15))
+  (fp_line (start 8.4 -10.92) (end 8.4 -4.45) (layer F.SilkS) (width 0.15))
+  (fp_line (start 8.4 14.78) (end -8.4 14.78) (layer F.SilkS) (width 0.15))
+  (fp_line (start -8.4 14.78) (end -8.4 -1.8) (layer F.SilkS) (width 0.15))
+  (fp_line (start -8.5 14.88) (end -8.5 -11.02) (layer F.Fab) (width 0.01))
+  (fp_line (start -8.4 -4.45) (end -8.4 -10.92) (layer F.SilkS) (width 0.15))
+  (fp_line (start 8.4 -1.8) (end 8.4 14.78) (layer F.SilkS) (width 0.15))
+  (fp_line (start -8.5 14.88) (end 8.5 14.88) (layer F.Fab) (width 0.01))
+  (fp_line (start 8.5 14.88) (end 8.5 -11.02) (layer F.Fab) (width 0.01))
+  (fp_line (start 8.5 -11.02) (end -8.5 -11.02) (layer F.Fab) (width 0.01))
+  (fp_line (start 9.6 -11.15) (end -9.6 -11.15) (layer F.CrtYd) (width 0.01))
+  (fp_line (start -9.6 15.15) (end -9.6 -11.15) (layer F.CrtYd) (width 0.01))
+  (fp_line (start -9.6 15.15) (end 9.6 15.15) (layer F.CrtYd) (width 0.01))
+  (fp_line (start 9.6 15.15) (end 9.6 -11.15) (layer F.CrtYd) (width 0.01))
+  (pad S thru_hole circle (at -8.065 -3.13) (size 2.54 2.54) (drill 1.57) (layers *.Cu *.Mask))
+  (pad S thru_hole circle (at 8.065 -3.13) (size 2.54 2.54) (drill 1.57) (layers *.Cu *.Mask))
+  (pad "" np_thru_hole circle (at -6.35 0) (size 3.25 3.25) (drill 3.25) (layers *.Cu *.Mask))
+  (pad "" np_thru_hole circle (at 6.35 0) (size 3.25 3.25) (drill 3.25) (layers *.Cu *.Mask))
+  (pad 1 thru_hole circle (at -5.075 11.18) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 2 thru_hole circle (at -3.045001 11.18) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 3 thru_hole circle (at -1.015002 11.18) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 4 thru_hole circle (at 1.014997 11.18) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 5 thru_hole circle (at 3.044996 11.18) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 6 thru_hole circle (at 5.074995 11.18) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 10 thru_hole circle (at 7.23 13.63) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 9 thru_hole circle (at 4.69 13.63) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 8 thru_hole circle (at -4.69 13.63) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
+  (pad 7 thru_hole circle (at -7.23 13.63) (size 1.2 1.2) (drill 0.89) (layers *.Cu *.Mask))
```

Comment:
These annular rings are too small - I'd go for something like 1.5 which will be much easier to solder and still leaves a lot of clearance.